CONTRIBUTING.md

```diff
@@ -1,108 +1,123 @@
+* Model is convertible by the `tools/downloader/converter.py` script. See [Model conversion](#model-conversion) for details.
+* Model is usable by demo or sample and provides adequate results. See [Demo](#demo) for details.
+* Model passes accuracy validation. See [Accuracy validation](#accuracy-validation) for details.
 
-Files location:
-* the configuration and documentation files must be in the `models/public/<model_name>` directory
-* the validation configuration file must be in the `tools/accuracy_checker/configs` directory
-* the demo must be in the `demos` directory
 
-This PR must pass next tests:
-* model is downloadable by `tools/downloader/downloader.py` script (see [Configuration file](#configuration-file) for details)
-* model is convertible by `tools/downloader/converter.py` script (see [Model conversion](#model-conversion) for details)
-* model can be used by demo or sample and provides adequate results (see [Demo](#demo) for details)
-* model passes accuracy validation (see [Accuracy validation](#accuracy-validation) for details)
+### PR Rejection
 
-At the end, your PR will be reviewed by OMZ maintainers for consistence and legal compliance.
+Your PR may be rejected in some cases, for example:
+* If a license is inappropriate (such as GPL-like licenses).
+* If a dataset is inaccessible.
+* If the PR fails one of the tests above.
 
-Your PR can be rejected in some cases, e.g.:
-* inappropriate license (e.g. GPL-like licenses)
-* inaccessible dataset
-* PR fails one of the test above
+## Configuration File
 
-## Configuration file
+The model configuration file contains information about model: what it is, how to download it, and how to convert it to the IR format. This information must be specified in the `model.yml` file that must be located in the model subfolder. 
 
-The model configuration file contains information about model: what it is, how to download it and how to convert it to IR format. This information must be specified in `model.yml` file, which must be located in the model subfolder. Let's look closer to the file content.
+Refer to the detailed descriptions of each file provided below.
 
 **`description`**
 
-Description of the model. Must match with the description from model [documentation](#documentation).
+Description of the model. Must match with the description from the model [documentation](#documentation).
 
 **`task_type`**
 
-Model task class, see [here](tools/downloader/README.md#model-information-dumper-usage) for details. If the task class of your model is absent, please add new to the list `KNOWN_TASK_TYPES` of the [tools/downloader/common.py](tools/downloader/common.py) file.
+Model task class, see [Model information dumper usage](tools/downloader/README.md#model-information-dumper-usage) for details. If there is no task class of your model, add a new one to the list `KNOWN_TASK_TYPES` of the [tools/downloader/common.py](tools/downloader/common.py) file.
```

Comment:
> see [Model information dumper usage](tools/downloader/README.md#model-information-dumper-usage)

it is better to use `here` or `in this file`, or whatever, but not `Model information dumper usage`

```suggestion
Model task class ( you can find list of task [here](tools/downloader/README.md#model-information-dumper-usage) in `task_type` element.
```

Reply:
It's the name of the file. It's better to tell user where we're pointing them to.

Reply:
I understand, but not in this case

Reply:
I guess I'll just make the _Model task class_ itself a link

Reply:
It is not well too. The point is that the link is not directly to model classes, user need to find them on this page. So, I prefer less explicit link description, like `for details`, or `additional info`.

*You can try it by yourself: open the link and seek `task_type` list element. That are model classes.*